.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,63 @@
+name: E2E Tests
+
+on:
+  push:
+    branches:
+      - trunk
+      - develop
+  pull_request:
+
+jobs:
+  E2ETests:
+    name: E2E Tests
+    runs-on: ubuntu-latest
+    env:
+      FORCE_COLOR: 2
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+            access_token: ${{ github.token }}
+
+      # Needed for Ubuntu 20.04.
+      - name: Kill mono to free 8084
+        run: |
+          sudo killall mono
```

Comment:
💅 
The port of e2e test site can be changed by [this config](https://github.com/woocommerce/google-listings-and-ads/blob/72669ef95ae6217b66f0c4e5b1359cf8c2252162/tests/e2e/config/default.json#L2).

Reply:
Yes, but then we'd have to update it locally, in the docs, and I'm afraid with the state of the WC e2e tooling, it may have some deeper hidden dependencies elsewhere. So I'd rather adapt the OS instance, to play with the defaults :see_no_evil: